### ISSUE TITLE
transparent mbtiles

### ISSUE THREAD

**Issue by moovida** [fleaflet/flutter_map#399]
Hi, I am trying to modify the MbTiles image provider to serve a transparent image where the mbtiles db doesn't have tiles. That way one could keep an online map layer below it and fill the surroundings with a nice context.

I am miserably failing to get transparency and after a while I thought that maybe there is something downstream that doesn't even allow it?

So, before going on in my search, the question is: Returning a transparent Codec tile image from the TileProvider should result in a transparent tile that would allow to view an underlying layer?

**Comment by johnpryan** [comment]
I think so - are you able to verify that the tiles are transparent? This package just uses standard image widgets in a stack, so I don't see a reason it wouldn't be working.

**Comment by johnpryan** [comment]
You might be able to use the DevTools to verify

**Comment by moovida** [comment]
Thanks for the reply. 
I made a first "to-be-sure" try. I created a transparent tile png and placed it in the assets.
Then I get it with 

```
        ByteData imageData = await rootBundle.load('assets/emptytile256.png');
        _emptyImageBytes = imageData.buffer.asUint8List();
```
and when the tile retrieval fails, I return the codec as:

```
return await PaintingBinding.instance.instantiateImageCodec(_emptyImageBytes);
```

It still doesn't work and now I have doubts. It this the right way to get an image or do I have to encode it? Outch, flutter is still mysterious  to me, even if beautiful :-)
